tools/ci_build/github/azure-pipelines/post-merge-jobs.yml

```diff
@@ -1,6 +1,100 @@
 stages:
+- stage: normal
+  dependsOn: [ ]
+  jobs:
+  - template: templates/c-api-linux-cpu.yml
+    parameters:
+      AdditionalBuildFlags: --use_extensions
+      BaseImage: 'centos:7'
+      OnnxruntimeArch: 'x64'
+      OnnxruntimeCFlags: '-Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -fstack-clash-protection -fcf-protection -O3 -Wl,--strip-all'
+      OnnxruntimeCXXFlags: '-Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -fstack-clash-protection -fcf-protection -O3 -Wl,--strip-all'
+      OnnxruntimeNodejsBindingArch: 'x64'
+      PoolName: 'Linux-CPU'
+
+  - template: templates/c-api-linux-cpu.yml
+    parameters:
+      AdditionalBuildFlags: --use_extensions
+      BaseImage: 'arm64v8/centos:7'
+      OnnxruntimeArch: 'aarch64'
+      OnnxruntimeCFlags: '-Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -fstack-clash-protection -O3 -Wl,--strip-all'
+      OnnxruntimeCXXFlags: '-Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -fstack-clash-protection -O3 -Wl,--strip-all'
+      OnnxruntimeNodejsBindingArch: 'arm64'
+      PoolName: 'aiinfra-linux-ARM64-CPU-2019'
```

Comment:
blank line

Reply:
could you please check why these post merge jobs workflows failed

Reply:
Someone uploaded the artifact without onnxruntime-extensions/archive/81e7799c69044c745239202085eb0a98f102937b.zip

Reply:
Lets get this PR merged first and than create a ticket for this new feature.